plugins/woocommerce/src/Blocks/BlockTypes/MiniCart.php

```diff
@@ -557,13 +578,72 @@ class="<?php echo esc_attr( $wrapper_classes ); ?>"
 	}
 
 	/**
-	 * Get the mini cart template part contents to render inside the drawer.
+	 * Remove innerHTML from WooCommerce blocks in the parsed blocks array.
+	 *
+	 * @param array $blocks The parsed blocks array.
+	 * @return array The filtered blocks array.
+	 */
+	protected function remove_unwanted_blocks( $blocks ) {
+		$filtered_blocks = array();
+
+		foreach ( $blocks as $block ) {
+			// Only remove outer wrapper from WooCommerce blocks.
+			if ( $this->is_woocommerce_template_block( $block ) ) {
+				$block['innerContent'] = $this->remove_outer_wrapper( $block['innerContent'] );
```

Comment:
I'm not really sure what `innerHTML` is used for, but maybe we should clear it too, because otherwise the divs will still be there.

Reply:
Indeed, it seems that after re-serializing the blocks, the content of `innerHTML` is empty the next time `parse_blocks` is used, so there's no need to modify it 👍